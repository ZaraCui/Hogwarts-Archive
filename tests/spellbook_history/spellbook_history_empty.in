ADD COLLECTION tests/data/books_small.csv
SPELLBOOK HISTORY 1004
EXIT

