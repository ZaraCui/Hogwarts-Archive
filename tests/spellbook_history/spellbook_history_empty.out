user: 12 spellbooks successfully added.

user: No rental history.

user: Ending Archive process.
